client/components/navbar.js

```diff
@@ -6,7 +6,7 @@ import {logout} from '../store'
 
```

Comment:
Just curious, I don't think we are using this anywhere right?

Reply:
not using nav bar but maybe we should?